backend/users/client.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from asgiref.sync import async_to_sync
+from django.conf import settings
+from pythonit_toolkit.service_client import ServiceClient
+
+GET_USERS_BY_IDS = """query GetUsersByIds($ids: [ID!]!) {
+    usersByIds(ids: $ids) {
+        id
+        displayName
```

Comment:
Can I use this query to get also the Gender of a user?

Reply:
yes I can add it